ocfweb/docs/docs/staff/backend/switch.md

```diff
@@ -39,14 +47,14 @@ blackhole.ocf.berkeley.edu>
 ```
 
 The switches can also be administered directly by connecting to their console port
-with a USB serial console cable.
+with a serial console cable.
 
-After logging in, one can enter an advanced configuration mode by typing "`enable`",
-and then, before configuring specific interfaces, type "`config`".
+After logging in, one must enter privileged exec mode by typing "`enable`",
+and then, before configuring specific interfaces, type "`config t`" or "`config terminal'".
```

Comment:
nit: line wrap (although this is pretty close to the right length anyway, so it probably doesn't matter either way)